tests/countries/test_democratic_republic_of_the_congo.py

```
@@ -0,0 +1,126 @@
+class TestDemocraticRepublicOfTheCongo(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(DemocraticRepublicOfTheCongo, years=range(1960, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(DemocraticRepublicOfTheCongo, CD, COD)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(DemocraticRepublicOfTheCongo(years=1959))
+
+    def test_2006(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)